Docs 8840 docs 8842 9623 dashboard fixes

sarathecoubian opened the pull request.

myuin-ms commented on modules/ROOT/pages/app-dashboards.adoc:
```diff
@@ -50,13 +50,13 @@ Built-in dashboards include various pages to view different types of charts:
+* Inbound - Response Time by Endpoint
+* Inbound - Total Requests by Endpoint
 * Inbound - Response Time graphs (99, 90, 75, and 50 percentiles)
-* Maximum Response Time
```
Take this line out. It's repeated below.

myuin-ms reviewed (commented):
In the "Outbound" tab, "ports" chart is listed twice. I realize it's an existing bug, let's fix it together now that I see it.

myuin-ms commented:
In addition, the performance section is still incorrect. There are still several inconsistencies between the doc and the actual product. If you need an account with data to see the actual charts, please ping the team.

myuin-ms commented:
Infrastructure charts are still off. What's documented does not match the product.

sarathecoubian commented on modules/ROOT/pages/app-dashboards.adoc:
```diff
@@ -50,13 +50,13 @@ Built-in dashboards include various pages to view different types of charts:
+* Inbound - Total Requests by Endpoint
 * Inbound - Response Time graphs (99, 90, 75, and 50 percentiles)
-* Maximum Response Time
+* Inbound - Maximum Response Time
```
```suggestion
```

sarathecoubian commented:
> Infrastructure charts are still off. What's documented does not match the product.

Updated. RE: Performance, the Outbound - Response Time Grouped by HTTP Endpoint should not have "grouped" in it. We removed that elsewhere.

dhirsch1 commented on modules/ROOT/pages/app-dashboards.adoc:
```diff
@@ -324,20 +331,47 @@ The sum of inbound requests whose response time is greater than 1000 ms, grouped
 
+Inbound - Response Time by Flow::
+The average, minimum, and maximum inbound response time, grouped by flow.
+* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
```
I don't think there should be a period here and elsewhere where there is similar text.

sarathecoubian replied:
```suggestion
* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms
```

sarathecoubian commented on modules/ROOT/pages/app-dashboards.adoc:
```diff
@@ -446,18 +480,18 @@ The average number of requests failures of connectors of type `other`.
-Inbound - Average Response Time Grouped by Endpoint::
+Inbound - Average Response Time by Endpoint::
 The average, maximum, and minimum response times, grouped by top occurring endpoint. For more information about which endpoints appear, see the section about xref:monitoring::app-dashboards.adoc#endpoint-operation-metrics[endpoint and operation data collection]. Cell color varies according to the following thresholds:
 * Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
 * Maximum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 500 ms; red if the response time is greater than or equal to 500 ms.
 * Minimum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
```
```suggestion
* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms
* Maximum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 500 ms; red if the response time is greater than or equal to 500 ms
* Minimum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms
```

sarathecoubian commented on modules/ROOT/pages/app-dashboards.adoc:
```diff
@@ -324,20 +331,47 @@ The sum of inbound requests whose response time is greater than 1000 ms, grouped
+Inbound - Response Time by Flow::
+The average, minimum, and maximum inbound response time, grouped by flow.
+* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
+* Maximum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 500 ms; red if the response time is greater than or equal to 500 ms.
+* Minimum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
```
```suggestion
* Maximum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 500 ms; red if the response time is greater than or equal to 500 ms
* Minimum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms
```

sarathecoubian commented on modules/ROOT/pages/app-dashboards.adoc:
```diff
@@ -324,20 +331,47 @@ The sum of inbound requests whose response time is greater than 1000 ms, grouped
+* Minimum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
+
+Outbound - Response Time by Flow::
+The average, minimum, and maximum outbound response time, grouped by flow.
+* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
+* Maximum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 500 ms; red if the response time is greater than or equal to 500 ms.
+* Minimum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
```
```suggestion
The average, minimum, and maximum outbound response time, grouped by flow.
* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms
* Maximum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 500 ms; red if the response time is greater than or equal to 500 ms
* Minimum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms
```
